admin.go

```
@@ -311,6 +312,13 @@ func handleViewAdminPage(app *App, u *User, w http.ResponseWriter, r *http.Reque
 		p.Content, err = getAboutPage(app)
 	} else if slug == "privacy" {
 		p.Content, err = getPrivacyPage(app)
+	} else if slug == "landing" {
+		p.Banner, err = getLandingBanner(app)
+		if err != nil {
+			return impart.HTTPError{http.StatusInternalServerError, fmt.Sprintf("Could not get banner: %v", err)}
+		}
+		p.Content, err = getLandingBody(app)
```

error is not being checked